lib/bolt_server/transport_app.rb

```diff
@@ -180,6 +180,21 @@ def run_script(target, body)
       [@executor.run_script(target, file_location, body['arguments'])]
     end
 
+    def in_pe_pal_env(environment)
+      if environment.nil?
+        [400, '`environment` is a required argument']
+      else
+        @pal_mutex.synchronize do
+          begin
+            pal = BoltServer::PE::PAL.new({}, environment)
+            yield pal
+          rescue Puppet::Environments::EnvironmentNotFound
+            [404, "Environment #{environment} not found"]
```

Comment:
I wasn't sure when I added these, but do we need to i18n these strings?

Reply:
Since bolt-server isn't technically documented as public API I don't think so. Nothing from a customer perspective should be routing directly to bolt-server: everything should come via orchestrator. So I think we can leave these as normal strings and let orch take care of i18n